data_tests/queries

```diff
@@ -0,0 +1,265 @@
+SELECT * FROM "h2o_feet" LIMIT 3'
```

Comment:
Whoops. Accidental commit. I'm just was keeping a log of all of the queries from the docs as a transitioned them into `.query` and `.json` files.